apps/web/src/components/quick-start/in-app-onboarding/TriggerNode.tsx

```diff
@@ -57,7 +57,10 @@ export function TriggerNode({ data }: { data: { label: string; email?: string }
 
 function TriggerButton({ setOpened }: { setOpened: (value: boolean) => void }) {
   const [notificationNumber, setNotificationNumber] = useState(1);
-  const { templates = [], loading: templatesLoading } = useTemplates();
+  const { templates = [], loading: templatesLoading } = useTemplates({
+    pageIndex: 0,
+    pageSize: 100,
+  });
```

Comment:
The templates are used here to check if the Onboarding Workflow already exists or not. Use same query cache.